Update readme.md with example docker compose (#130)
macbre/docker-nginx-http3: readme.md

docker pull ghcr.io/macbre/nginx-http3:latest
DOCKER_BUILDKIT=1 docker build . -t macbre/nginx --cache-from=ghcr.io/macbre/nginx-http3:latest --progress=plain
```

> Docker Compose example ,  it is necessary to expose through UDP as well as TCP to obtain HTTP/3

Reviewer: ```suggestion
### Docker Compose example

It is necessary to expose both UDP and TCP ports to be able to HTTP/3
```

> ```

Reviewer: ```suggestion
```yaml
```

  nginx:
    image: macbre/nginx-http3
    ports:
      - '443:443/tcp'
      - '443:443/udp' # use UDP for usage of HTTP/3
```

> Note : ports in TCP and UDP to expose HTTP/3 they must be the same

Reviewer: ```suggestion
Note: both TCP and UDP HTTP/3 ports needs to be the same
```
